### PR TITLE
Pin cftime at specific places for the docs

### PR DESCRIPTION
**Pull request by aulemahal**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes all the darned doc failures due to nc-time-axis.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated (with summary of main changes)
- [ ] `bumpversion (major / minor / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)

* **What kind of change does this PR introduce?** 
Piins cftime on read the docs and tox's doctests by re-installing 1.4.1 after the main package installation part.

* **Does this PR introduce a breaking change?** 
Nope.

* **Other information**:

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -37,6 +37,7 @@ deps =
     mypy
 setenv = PYTEST_ADDOPTS = "--color=yes"
 commands =
+    pip install cftime==1.4.1
```
Comment (Zeitsperre) [inline tox.ini:40]: Why not add this to the `deps` section?

**Review by Zeitsperre — APPROVED** [review]
Looks good!

File: tox.ini (hunk 1 above)
Comment (aulemahal) [reply tox.ini:40]: My goal was to limit this as much as possible. Xclim works with cftime 1.5.0, so no need to pin it, it's not "their" fault.

Comment (Zeitsperre) [reply tox.ini:40]: I just meant you could have added `cf-time==1.4.1` to the deps section of the `doctest` overrides a few lines above. Both work though.

**Pull request merged**